.typos.toml

```diff
@@ -0,0 +1,40 @@
+[files]
+extend-exclude = ["docs/versioned"] # don't check ./docs/versioned
+
+[default]
+locale = "en-us"
+
+[default.extend-identifiers]
+FilterThreshhold = "FilterThreshhold" # ignore error, it is actually "Paper.FilterThreshhold" in Paper
```

Comment:
Can it be ignored per-file instead? Incase someone else types it not refering to the option?

It's quite a narrow case tho

Reply:
I'm afraid not, there'd have to be a separate config